tests/snapshot/conftest.py

```diff
@@ -22,15 +22,24 @@ def _make_vdi(name):
 @pytest.fixture(scope='module')
 def vm_with_vbds(host, vdis, imported_vm):
     def _make_vbd(host, device, vdi):
-        host.xe('vbd-create', {
+        return host.xe('vbd-create', {
             'vm-uuid': vm.uuid, 'mode': 'RW', 'type': 'Disk', 'device': device, 'vdi-uuid': vdi
         })
 
+    def _remove_vbd(host, vbd_uuid):
```

Comment:
I don't understand why we use local functions with a "_" prefix.
Also, why do we use local functions that are called only once instead of directly executing block code?